config/default.py

```diff
@@ -72,7 +72,7 @@
 JOBS_SECONDS_BETWEEN_PENDING_CHECK = 60
 
 # Logging
-LOGGING_FORMAT = '[%(asctime)s] - %(levelname)s: %(message)s [in %(pathname)s:%(lineno)d]'
+LOGGING_FORMAT = '[%(asctime)s] - %(levelname)s: %(message).10000s [in %(pathname)s:%(lineno)d]'
```

Comment:
The Canvas API client likes to log the whole API response at debug level, which in the case of CSV provisioning reports amounts to logging the whole CSV, which is dumb. We don't have a good way to get into the guts of the API client, so this formatter just truncates the whole log message at a generous 10,000 characters.